website/docs/d/pod.html.markdown

```diff
@@ -310,8 +310,8 @@ Please, look at the example below:
 
 ```hcl
 resources {
-    limits = {}
-    requests = {}
+  limits   = {}
```

Comment:
This and the rest of the `website/docs` changes are not related to the original issue, just fix for some issues found by linter.